treeherder/webapp/api/logslice.py

```diff
@@ -77,16 +67,16 @@ def list(self, request, project, jm):
 
         try:
             gz_file = filesystem.get(url)
-
             if not gz_file:
-                handle = self.get_log_handle(url)
-                gz_file = gzip.GzipFile(fileobj=BytesIO(handle.read()))
+                r = requests.get(url,
```

Comment:
Let's use `make_request()` since it also sets a timeout, calls `raise_for_status()` and avoids the user agent duplication :-)